frontend/.eslintrc.js

```diff
@@ -1,13 +1,11 @@
 "use strict";
 
 module.exports = {
-  extends: "@adfinis-sygroup/eslint-config/ember-app",
-
+  settings: {
+    "import/internal-regex": "^customer-center/",
+  },
+  extends: ["@adfinis/eslint-config/ember-addon"],
```

Comment:
```suggestion
  extends: ["@adfinis/eslint-config/ember-app"],
```

It's an `app`, not a `addon`.

Reply:
Done in https://github.com/adfinis/customer-center/pull/412/commits/179426966a197d6f2126c01d5767b25b74542d4f